pjsip/src/pjsip/sip_parser.c

```diff
@@ -1783,6 +1789,13 @@ static void parse_generic_array_hdr( pjsip_generic_array_hdr *hdr,
 }
 
 /* Parse generic array header. */
+static void parse_generic_array_hdr( pjsip_generic_array_hdr *hdr,
```

Comment:
There’s no value in having parse_generic_array_hdr and pjsip_parse_generic_array_hdr_imp. Delete on of them – probably parse_generic_array_hdr, and update the references to it.